R/utils.R

```diff
@@ -1,12 +1,18 @@
-test_integration <- function (input = "integration-test.Rmd") {
+test_integration <- function(input = "integration-test.Rmd") {
   needless <- tempfile(fileext = ".html")
   rmarkdown::render(input, output_file = needless)
 
   invisible(input)
 }
 
 setup_project <- function() {
-  source(fs::path(
-      "dry", "set_portfolio-name-ref-all_working-location_and_web-parameters.R"
-  ))
+  path <- fs::path(
+    "deduplicate",
+    "set_portfolio-name-ref-all_working-location_and_web-parameters.R"
+  )
+  source(path)
+}
+
+use_r_packages <- function(path = "deduplicate/load-and-attach-r-packages.R") {
+  source(path)
 }
```

Comment:
This helper hides the details about where and how we load and attach packages -- which are a low level detail that is likely to change. This way the change will happen in just one place.